independent-publisher-connectors/Exact Online Time & Billing/readme.md

```diff
@@ -0,0 +1,148 @@
+# Exact Online - Time & Billing
+Exact Online covers both accounting and CRM, providing firm foundations on which to build a strong company.
+This connector contains the methods that can be used with the Exact Online module "Time & Billing".
+
+## Publisher: Indocs
+
+## Prerequisites
+You need an [Exact Online account](https://www.exact.com/try).
```

Comment:
@Knots23 We expect partners to provide each and every piece of information in the readme.md file to obtain credentials. You are just redirecting us to the product website to get an account. We need to set up an app with the test account we going to create, and the product is not offering any free account. Even if we get a trial account, it would be expired in the next 30 days and the OAuth app we need to set up for your connector would not work.

![image](https://user-images.githubusercontent.com/56276394/172741789-2866b4bb-a7fa-43d3-80c6-2d7cb153fefd.png)

@natalie-pienkowska It is a paid service and the test account we going to create would expire in the next 30 days, and the OAuth app would not work and we may receive failure alerts. Can you please let me know if we can move forward with this connector that has paid service/product?

Reply:
@sriyen-msft Sorry it took a while: work.
The prerequisites are that you have an Exact Online account, yes. Is this the first IP connector that connects to a paid service?
The part describing how to obtain the credentials are in the readme, in the section below. I pointed to their documentation, as it may change in the future. If you want, I can copy-paste it directly into the readme.